lib/Doctrine/Migrations/Metadata/AvailableMigrationsSet.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Metadata;
+
+use Countable;
+use Doctrine\Migrations\Exception\MigrationNotAvailable;
+use Doctrine\Migrations\Version\Version;
+use function array_values;
+use function count;
+
+/**
+ * Represents a non sorted list of migrations that may or may not be already executed.
+ */
+final class AvailableMigrationsSet implements Countable
+{
+    /** @var AvailableMigration[] */
```

Comment:
Alternative suggested [here](https://twitter.com/Ocramius/status/1258360055151439872):

```suggestion
    /**
     * @var AvailableMigration[]
     * @phpstan-var AvailableMigration[]
     * @psalm-var AvailableMigration[]
     */
```

Not that the second one is superflous: both tools read each other's annotations